docs/design/README.md

```diff
@@ -2,66 +2,65 @@
-)
-   * [Mutation Interfaces (for the current approach)](https://docs.google.com/document/d/1c5Z3g6Zsfmga7xod4--t6RrXamdAaMaqYzPNujtZb10/edit)
-   * [OPA Gatekeeper Mutation (alternate proposal)](https://docs.google.com/document/d/1LtNIxd-Y3oh0HPgPYdaVV7t2c6vBSwzYAFcTyxQYZjE/edit)
-   * [Mutation Revised Design Doc (June 2019)](https://docs.google.com/document/d/1G7WgZKx1Y3VOTUjrqn7DjDaZgSKCIZowILm_I6psrw0/edit#heading=h.mtvdjag5uj9)
-   * [Mutation Initial Design Doc (April 2019)](https://docs.google.com/document/d/1qTHwqoUX8AL2jodyWKB_2szrGDwhi14Ra_LlQ-ogtck/edit#heading=h.iu1ppjy7g7j)
-* [External Data](https://docs.google.com/document/d/1hPi86jdsCKg8puYT5_s_73mPGExUJeZfyKmvG-XWtPc/edit#)
-* [gator validate](https://docs.google.com/document/d/1B0hXDia8SExOkCVAEbPVHFJmnWeJgHziGvMxnzrENa0/edit)
-* [External Data TLS Support](https://docs.google.com/document/d/1GjV3WeC2bgQ3j37_mMpY9hr7YOAqzSJ6jDSu-DVrcmU/edit)
+* [Gatekeeper v1beta1 CRD Deprecation](https://docs.google.com/document/d/1quP6ScoPiKDJrykOaReP8xRFdl66LO0586P_v5lfnQY/edit?usp=drive_link)
+* [Compiler Sharding Design](https://docs.google.com/document/d/1CV9HCp8TG-q_B6TltufHIkXcTrb03LTPWzD2Vr4R_ec/edit?usp=drive_link)
+* [Template e2e Testing Design](https://docs.google.com/document/d/1_DDhxDYjVxyZk090F-GznHTeT5jFMpwjEXLLhzR3AoU/edit?usp=drive_link)
+* [ByPod Status Design](https://docs.google.com/document/d/1kPb3B1I6FBsthpR6hp0Q3SEB11iXoSSuWbj9t1q-_58/edit?usp=drive_link)
+* [Mutation Design - 4th Edition (current approach)](https://docs.google.com/document/d/1FGJ8A9_JnGJu-pU0u6hh37JKWw7j7GhMfrvTFL9sfUA/edit?usp=drive_link)
+   * [Mutation Interfaces (for the current approach)](https://docs.google.com/document/d/1ihhUKly1YFJfvmTKjJLAOkQlrM1Oexr147IJbSHu5Ao/edit?usp=drive_link)
+   * [OPA Gatekeeper Mutation (alternate proposal)](https://docs.google.com/document/d/11V3zfQ75ugxUy_I2o6AqjWbhw1PKcMh8v3Cf8_uuwdU/edit?usp=drive_link)
+   * [Mutation Revised Design Doc (June 2019)](https://docs.google.com/document/d/1jluk3xL3dLxsE_9C69XWVOrSyLqzsySY0W7r9Yl0i68/edit?usp=drive_link&resourcekey=0-A4jzI3xwwxfggM9uvwctAw)
+   * [Mutation Initial Design Doc (April 2019)](https://docs.google.com/document/d/1-TRvdND3Q-HPzIg8pe2ZtYNkOBDnQ2CxC7SWhA0lP6g/edit?usp=drive_link)
+* [External Data](https://docs.google.com/document/d/1tXsoXWFzoCdWY8BqxXCNr5qXkTbGgEf2-sEpvGSE4ZQ/edit?usp=drive_link)
+* [gator validate](https://docs.google.com/document/d/1LG-GfOlXjWzz0wNCb8jdzIMplZqkyrkMmTLYx05CKY4/edit?usp=drive_link)
+* [External Data TLS Support](https://docs.google.com/document/d/1z43LI38twxPiUHPuktWPB1xMnsYr7SaVgczq-WsBhck/edit?usp=drive_link)
+* [Export violations using pub-sub](https://docs.google.com/document/d/1xu6c99m_qBOpztAc8uUnoY6ST8UiXyyJzrXOmMmJQ9I/edit?usp=drive_link)
 
 ## Roadmap (in development)
 * See [milestones](https://github.com/open-policy-agent/gatekeeper/milestones?direction=asc&sort=due_date)
 
 ## Roadmap (complete)
```

Comment:
Perhaps we can remove this section and move all the links under `Implemented` since this section was originally added for GA and post GA roadmap.